modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -1,15 +1,15 @@
 [id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
 = Configuring JSON log data for Elasticsearch
 
-When forwarding JSON logs to an Elasticsearch log store, you must create an index for each format if the JSON log entries _have different formats_.
+If your JSON logs have more than one schema, storing them in a single index might cause type conflicts and cardinality problems. To avoid that, you must configure the `ClusterLogForwarder` custom resource (CR) to group each schema into a single output definition. This way, each schema is forwarded to a separate index.
+
+If you forward to Red Hat's OpenShift Logging managed Elasticsearch, it will try and generate a new index based on your configuration.
```

Comment:
Should that be captured into a "[NOTE]" tag?

Reply:
Updated. Thanks.